editoast/editoast_core/src/lib.rs

```diff
@@ -113,27 +113,32 @@ impl CoreClient {
 /// A struct implementing this trait represents a Core request payload
 ///
 /// For example:
-/// The example is marked `ignore` because `await` requires
-/// an async runtime (e.g., Tokio) and proper async context to execute.
-/// ```ignore
+/// ```
+/// # use editoast_core::AsCoreRequest;
+/// # use editoast_core::Bytes;
+/// # use serde::Serialize;
+/// # use serde::Deserialize;
 /// #[derive(Serialize, Default)]
 /// struct TestReq {
 ///     foo: String,
 ///     bar: i64
 /// }
 ///
-/// #[derive(Deserialize)]
-/// struct Response {
-///     message: String
-/// }
-///
-/// impl AsCoreRequest<Response> for TestReq {
+/// impl AsCoreRequest<Bytes> for TestReq {
 ///    const METHOD: reqwest::Method = reqwest::Method::POST;
-///    const URL_PATH: &'static str = "test01";
+///    const URL_PATH: &'static str = "/some/path";
+/// #  fn infra_id(&self) -> std::option::Option<i64> { Some(42) }
```

Comment:
```suggestion
///  fn infra_id(&self) -> std::option::Option<i64> { Some(42) }
```

Not sure if it's necessary to hide it since it's a required implementation anyway

Reply:
Done in [73d4a7c](https://github.com/OpenRailAssociation/osrd/compare/27fd93d74cb293dbf11e14b9344bd4fb8bfcbf27..73d4a7c77117569ec30e4a84b7476ec993cb56ce).